src/modules/commentsApi.js

```diff
@@ -1,4 +1,4 @@
-const appId = 'QALIoXYe7LSsEgG2FYGA/';
+const appId = 'f5fDagMARo6BC6LKqLaP/';
```

Comment:
- [ ] [OPTIONAL] When I click on close popup button (X) its refreshing the page 👌🏼. Please kindly fix this issue